react-insta-clone/src/components/PostContainer/PostContent/NewComment/NewComment.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './NewComment.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const NewComment = (props) => {
+  return (
+    <form className='comment-section' onSubmit={props.onSubmit}>
+        <input placeholder='Add a comment...' value={props.value} onChange={props.onChange}></input>
```

Comment:
Again, make sure to spread the properties out in different lines as below:
```js
<input
  placeholder="Add a comment... "
  value={props.value}
  onChange={props.onChange}
/>
```
Also try to use self-closing tag when you can. In jsx and html, it's more common to use double quotes rather than single quotes. Single quotes are recommended for strict JS like when you are importing in the beginning of your code:
```js
import React from 'react';
```